.kokoro/build-java.sh

```diff
@@ -1,36 +0,0 @@
-#!/bin/bash
```

Comment:
It appears there are other repos that use dpebot, such as java-repo-tools. We don't want to remove this file while those configurations are still active.

Reply:
https://github.com/GoogleCloudPlatform/java-repo-tools and https://github.com/GoogleCloudPlatform/getting-started-java have migrated to renovate.